spaceship/lib/spaceship/tunes/app_submission.rb

```diff
@@ -132,6 +131,15 @@ def complete!
         end
         raw_data_clone.delete("version")
 
+        # Check whether the application makes use of IDFA or not
+        # and automatically set the mandatory limitsTracking value in the request JSON accordingly.
+        if !self.add_id_info_uses_idfa.nil? && self.add_id_info_uses_idfa == true
+          raw_data_clone.set(
```

Comment:
Thanks for the comment @minuscorp. Sorry I should have been more clear, could you put a comment above this line to explain what this method is doing?

Reply:
My bad 😅